pkg/openapi/server.spec.yaml

```diff
@@ -2465,6 +2528,92 @@ components:
                 value: 9361402c-f998-49cc-ab21-9bb99afcfde8
             spec:
               regionId: c7568e2d-f9ab-453d-9a3a-51375f78426b
+    imageCreateRequest:
+      description: A request for an image.
+      content:
+        application/json:
+          schema:
```

Comment:
Usually I have an aversion to inline schemas, as if we need to use it again it needs moving to the schema section where it should have been from the beginning.  In this case most of the stuff is creation time only so you may get away with it.  The one caveat to that is the metadata... you can envision a world where the end user wants to update the image name.